test/common/include/alpaka/test/KernelExecutionFixture.hpp

```diff
@@ -53,7 +53,7 @@ namespace alpaka
                 typename... TArgs>
             auto operator()(
                 TKernelFnObj const & kernelFnObj,
-                TArgs const & ... args)
+                TArgs && ... args)
```

Comment:
You probably want to use `std::forward` at the place where `args` is used within this method.